routes/newProductSubmission.js

```diff
@@ -0,0 +1,85 @@
+ var mysql = require("mysql");
+ var config = require("konfig")();
+ var express = require("express");
+ var router = express.Router();
+ /* Is there a way to do this without the relative path? */
+ var db = require("../imp_modules/impdb.js");
```

Comment:
I will add a module that allows us to remove the need for relative path, but for now this works.